.mypy.ini

```diff
@@ -2,3 +2,4 @@
 follow_imports = skip
 ignore_missing_imports = True
 scripts_are_modules = True
+; platform = darwin
```

Comment:
Leftover from testing ?

Reply:
left there (this and the one below) are commented out; but left there intentionally to allow for testing